test/integration/features/addons/addon-htpasswd-identity-provider.feature

```diff
@@ -0,0 +1,86 @@
+@addon-htpasswd-identity-provider @core
```

Comment:
Tag ```core``` is used only for testing propose through pr. Should be removed before merge

Reply:
why is it not possible to remove now?

Reply:
Will remove it for sure after first or second round of review comment implementation is tested in CI through this pr